### ISSUE TITLE
Admin console "CONNECT" command to open new telnet window

### ISSUE THREAD

**Issue by DavidRieman** [WheelMud/WheelMUD#69]
Usually after we spin up the game instance, we want to follow that up by connecting a telnet client to the game instance.  It would be easier if the admin console gave us a basic "connect" option.  When run, it would try to see if we have a "telnet.exe" available, or perhaps other clients (configurable via app.config or via multiple connection commands?) to open with an automatic localhost connection to the correct port.  This way a developer can opt to close the telnet app/window each time they're done with it instead of having to juggle the extra window while they don't need it.

**Comment by duaneking** [comment]
I think anything that is admin related needs security.

I think requiring the admin connect to unlock the instance on boot is going to be a bad idea when serves need to reboot and admins cant connect to unlock them.

**Comment by DavidRieman** [comment]
@duaneking I think maybe I didn't type this issue very clearly.  In migrating it to the base repo (https://github.com/DavidRieman/WheelMUD/issues/46), I've tried to adjust a bit of wording to hopefully clear it up.

I think your interpretation was that I wanted a "connect" command to require the admin giving this command before telnet connections would be accepted.  Agreed, I don't feel that's a good idea in general.  On the other hand, I have filed https://github.com/DavidRieman/WheelMUD/issues/47 for an opt-in to something similar, for emergency debugging purposes.

If I've misinterpreted wildly and there's an unresolved issue between these, feel free to open up a [discussion](https://github.com/DavidRieman/WheelMUD/discussions) or whatnot.  Thanks!
